README.md

```diff
@@ -82,20 +79,33 @@ None
 
 *Note: For ansible-core 2.18, CI only covers sanity tests and no integration tests will be run until the test environment is released.*
 
-## Roadmap
+## Contributing
+
+We welcome community contributions to this collection. For more details, see [Contributing to Ansible-maintained collections](https://docs.ansible.com/ansible/devel/community/contributing_maintained_collections.html#contributing-maintained-collections) for complete details.
+
+* [Issues](https://github.com/ansible-collections/ansible.posix/issues)
+* [Pull Requests](https://github.com/ansible-collections/ansible.posix/pulls)
+* [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html)
+
+
+## Support
 
-<!-- Optional. Include the roadmap for this collection, and the proposed release/versioning strategy so users can anticipate the upgrade/update cycle. -->
+* Join the Ansible forum:
+  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
+  * [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+  * [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
```

Comment:
```suggestion
* Join the Ansible forum:
  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
  * [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
  * [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
```
I believe the communication section should stay at the beginning of the file untouched as it's more about just support. It was put there because of its importance. If this support section is necessary, I would just refer from it to that Communication section.

That readme template by RH should be changed, I added it to a related jira task https://issues.redhat.com/browse/AAP-35113 I'm going to do soon

The rest LGTM

Reply:
Indeed. I will do :)

1. Create `Communication` section on the top of README
2. Move the contents from `Support` to `Communication` section
3. I will leave an anchor link to the Communication section in the Support section.